### PR TITLE
 Deprecate support for the v1 collection format

### PR DESCRIPTION
**Pull request by codenirvana**
Reverts #1660

### CODE REVIEW

File: lib/run/options.js [hunk 1]
```diff
@@ -65,21 +74,25 @@ var _ = require('lodash'),
...
      */
-    validateCollection = function (collection, callback) {
-        var error = null;
-
+    processCollection = function (collection, callback) {
         if (util.isV1Collection(collection)) {
-            error = new Error('Newman >= v4 does not support the v1 collection format');
-            error.friendly = 'Use the Postman Native app to export collections in the v2 format';
+            // @todo: route this via print module to respect silent flags
+            console.warn('newman warning: Newman >= v4 deprecates support for the v1 collection format');
```
Comment (kunagpal) [inline lib/run/options.js]: `newman:`

Comment (kunagpal) [reply lib/run/options.js]: `Newman v4`

File: test/unit/run.test.js [hunk 2]
```diff
@@ -31,36 +31,6 @@ describe('run module', function () {
         }).to.not.throw();
     });
 
-    it('should error out if v1 collection is passed', function (done) {
```
Comment (kunagpal) [inline test/unit/run.test.js]: Skip this test with a todo instead of removing it altogether.

File: lib/run/index.js [hunk 3]
```diff
@@ -54,8 +54,8 @@ var fs = require('fs'),
      * @type {Object}
      */
     knownReporterErrorMessages = {
-        html: '  run `npm install newman-reporter-html`',
-        teamcity: '  run `npm install newman-reporter-teamcity`'
+        html: '  run `npm install newman-reporter-html`\n',
```
Comment (shamasis) [inline lib/run/index.js]: lol ... i spotted this sneaky out of context change in this PR. don't make a habit of it. :-p

**Pull request merged**